github/data_source_github_repository_test.go

```diff
@@ -55,29 +55,35 @@ func TestAccGithubRepositoryDataSource(t *testing.T) {
 
 		config := map[string]string{
 			"no_match_name": `
-				data "github_repository" "no_match_name" {
+				data "github_repository" "test" {
```

Comment:
Do you mind explaining the reasoning behind this change?

Reply:
It's been too long (sorry about that) but I think it was because of the following assertion. I could only find how to create one check, so I named both configurations the same. If that makes sense?

Although, now that I read through it and see your other comment, I think I need to get a better idea of how this testing framework actually works. :)